deploy/infrastructure/dev/us-east-2/.terraform.lock.hcl

```diff
@@ -3,8 +3,9 @@
 
 provider "registry.terraform.io/hashicorp/aws" {
   version     = "4.7.0"
-  constraints = ">= 3.63.0, >= 3.72.0"
+  constraints = ">= 2.23.0, >= 2.49.0, >= 2.53.0, >= 3.28.0, >= 3.63.0, >= 3.72.0"
```

Comment:
which version of terraform are you using locally?

Reply:
```
terraform version
Terraform v1.3.6
on darwin_arm64
+ provider registry.terraform.io/hashicorp/aws v4.7.0
+ provider registry.terraform.io/hashicorp/cloudinit v2.2.0
+ provider registry.terraform.io/hashicorp/kubernetes v2.10.0
+ provider registry.terraform.io/hashicorp/tls v3.1.0
```

Reply:
I can't reproduce this lock file change locally but the changes seem harmless.

Reply:
Did you have a pre-existing `.terraform` folder which was created by an old version of `terraform`? 

Can I ask you to try deleting `.terraform` on your machine, pull the lock from `main` and see if running `terraform init` changes the lock again please?